nlu/models/intent_classifier.py

```diff
@@ -34,6 +36,7 @@
     'fr': "dbmdz/bert-base-french-europeana-cased",
 }
 
+mlflow.set_tracking_uri("http://0.0.0.0:5002")
```

Comment:
```suggestion
mlflow.set_tracking_uri("http://0.0.0.0:5002")
```
This should be the docker hostname